### DISCUSSION TITLE
alternative way to use run nhost locally on windows

### DISCUSSION THREAD

**Discussion by joaomelo** [nhost/nhost#1117] [Q&A] [answered]
Hi there,

I understand that the way to use the NHost client on a windows machine is through the linux subsystem. I don't question this solution as a legitimate approach, but it's suboptimal for my context.

Concerning only the ability to execute NHost locally for manual/auto testing, Is there an alternative approach on Windows?

**Comment by elitan** [comment, marked as answer]
Not at this moment, but feel free to create an issue on our CLI repo. We'll tackle this eventually if not a community member is faster.

http://github.com/nhost/cli

**Reply by joaomelo** [reply]
Will do @elitan. Tks!

**Reply by joaomelo** [reply]
for an eventual future searcher, here is the issue: https://github.com/nhost/cli/issues/439
